spec/requests/api/v0/articles_spec.rb

```diff
@@ -45,7 +45,7 @@
     context "without params" do
       it "returns json response" do
         get api_articles_path
-        expect(response.content_type).to eq("application/json")
+        expect(response.media_type).to eq("application/json")
```

Comment:
`content_type` will contain the full content type (with encoding and other options) in Rails 6.1, so 6.0 signals this access with a deprecation and suggests to move to `media_type` to check only that